superset-frontend/src/explore/actions/exploreActions.ts

```diff
@@ -86,7 +86,7 @@ export function fetchFaveStar(sliceId: string) {
 
 export const SAVE_FAVE_STAR = 'SAVE_FAVE_STAR';
 export function saveFaveStar(sliceId: string, isStarred: boolean) {
-  return function (dispatch: Dispatch<ReturnType<typeof addDangerToast>>) {
```

Comment:
The latest version of Redux has a better-typed `dispatch` function, so the extra typing here doesn't compile anymore, for good reason.